src/category_theory/arrow/limit.lean

```diff
@@ -0,0 +1,63 @@
+import category_theory.arrow.basic
+import category_theory.limits.has_limits
+
+namespace category_theory
+
+namespace arrow
+
+universes v u
+
+variables {C : Type u} [category.{v} C]
+
+/-- Make a limit cone for a diagram of arrows, given limit cones for the left and right. -/
+def limit_cone {J : Type v} [small_category J] (F : J ⥤ arrow C)
+  (CL : limits.limit_cone (F ⋙ left_func))
+  (CR : limits.limit_cone (F ⋙ right_func)) :
+  limits.limit_cone F :=
+{ cone :=
```

Comment:
Could you separate the construction of the cone itself?
I'm thinking of a similar structure to what's done in `category_theory/limits/functor_category.lean`.